src/test/java/uk/gov/ons/ctp/response/collection/exercise/service/EventServiceTest.java

```diff
@@ -490,6 +491,60 @@ public void testProcessEventsOnlyEventInFuture() {
     verify(actionSvcClient, never()).processEvent(any(), any());
   }
 
+  @Test
+  public void testEventStatusNothingToUpdate() {
+    // Given
+    List<Event> list = new ArrayList<>();
+    Event event = createEvent(Tag.mps, "31/12/2999");
```

Comment:
general question. Is there such thing as a max date or value

Reply:
or could this be  defined as a static final as it is used in other places

Reply:
Ideally the dates should have been different but it doesn't make any difference to check the logic. Yes we can but I think we can keep it as it is for now.